src/pipeline.py

```diff
@@ -26,13 +26,26 @@ class Pipeline:
         The processors to apply, in order.
     exclude_labels : list
         List of labels to exclude from processing.
+    ahn_reader : AHNReader object
+        Pointer to the AHNReader object used in the Processors. Required if
+        caching is used.
+    caching : bool (default: True)
+        Enable caching of AHN interpolation data.
     """
 
     FILE_TYPES = ('.LAS', '.las', '.LAZ', '.laz')
 
-    def __init__(self, process_sequence=[], exclude_labels=[]):
-        self.process_sequence = process_sequence
+    def __init__(self, processors=[], exclude_labels=[],
+                 ahn_reader=None, caching=True):
```

Comment:
Zet caching maar op False. Dan hoeven we de example notebooks minimaal aan te passen